scss/tech.scss

```diff
@@ -22,14 +21,19 @@ $tech-logo-size: 100px;
   position: relative;
   width: 100%;
 
+  &:hover::before {
+    top: 5px;
+    left: 5px;
```

Comment:
Remember this card is meant to be a transition, so you should add it as well as the transition you are doing. Use the `transition` property for that purpose 😄